redis/apiv1beta1/cloud_redis_client.go

```diff
@@ -117,7 +117,7 @@ type CloudRedisClient struct {
 //   As such, Redis instances are resources of the form:
 //   /projects/{project_id}/locations/{location_id}/instances/{instance_id}
 //
-// Note that location_id must be refering to a GCP region; for example:
+// Note that location_id must be referring to a GCP region; for example:
```

Comment:
This file is auto-generated from [this proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/redis/v1beta1/cloud_redis.proto#L45). I'll look into getting that fixed internally. Will you revert this file for this PR?

Reply:
Thanks tbpg! Already reverted this file.